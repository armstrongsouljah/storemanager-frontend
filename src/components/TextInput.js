import React from "react";

const TextInput = props => (
    <div className="input-field">
        <input id={props.id} type={props.type} className="validate" onChange={props.onChange} />
        <label htmlFor={props.id}>{props.labelText}</label>
  </div>
);

export default TextInput;
