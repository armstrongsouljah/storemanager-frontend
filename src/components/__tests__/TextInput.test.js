import React from "react";
import { shallow, } from "enzyme";
import TextInput from "../TextInput";

const props = {
    onChange: jest.fn(),
    id: "",
    labelText: "",
    type: "",
};

describe("<TextInput/>", () => {
    it("Should render without crashing", () => {
        const wrapper = shallow(<TextInput {...props} />);
        expect(wrapper).toMatchSnapshot();
    });
});
