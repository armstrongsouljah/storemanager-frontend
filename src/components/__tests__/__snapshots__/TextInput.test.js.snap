// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TextInput/> Should render without crashing 1`] = `ShallowWrapper {}`;
